LICENSE

```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```

Comment:
We decided a few months ago that we use Apache 2 for all public open source projects. I copied the full license text instead of the "shorter" version that includes a link.

Changed it to what we use in other projects too.